lib/bolt/module_installer/specs/git_spec.rb

```diff
@@ -11,7 +11,7 @@ module Bolt
   class ModuleInstaller
     class Specs
       class GitSpec
-        NAME_REGEX    = %r{\A(?:[a-z][a-z0-9_]*[-/])?(?<name>[a-z][a-z0-9_]*)\z}.freeze
+        NAME_REGEX    = %r{\A(?:[a-zA-Z0-9]+[-/])?(?<name>[a-z][a-z0-9_]*)\z}.freeze
```

Comment:
I'm not really sure if this should be the same as the Forge specs, or if it should be a bit more permissive:

```
\A(?:\w+[-/])?(?<name>[a-z][a-z0-9_]*)\z
```

Reply:
This works just fine - git urls aren't case sensitive.